deploy/helm/templates/_helpers.tpl

```diff
@@ -311,3 +312,14 @@ Define default storage class name, if cloud provider is known, specify a default
 {{- "" }}
 {{- end }}
 {{- end }}
+
+{{/*
+defaultProvider could be "aws","gcp","aliyun","tencentCloud", "huaweiCloud", "azure"
+*/}}
+{{- define "kubeblocks.defaultProvider" -}}
+{{- if or (eq .Values.provider "aws") (eq .Values.provider "gcp") (eq .Values.provider "aliyun") (eq .Values.provider "tencentCloud") (eq .Values.provider "huaweiCloud") (eq .Values.provider "azure") }}
```

Comment:
Prefer to define a validProviders slice, and travel the slice and check.

Reply:
> Prefer to define a validProviders slice, and travel the slice and check.

Make a validate like:
![image](https://github.com/apecloud/kubeblocks/assets/101848970/76d057c4-2753-4660-b97f-d7df14acffa9)